src/storage/mvcc/txn.rs

```diff
@@ -60,6 +60,11 @@ impl<S: Snapshot> MvccTxn<S> {
         })
     }
 
+    // We may set scan mode Forward to skip seek key which not existed
+    pub fn set_scan_mode(&mut self, mode: ScanMode) {
+        self.reader.set_scan_mode(mode);
+    }
+
```

Comment:
This setter function makes it looks like it can be changed any time, but actually it can't. I suggest you don't do that to avoid potential misuse in the future. Please consider move the ScanMode to a parameter of `MvccTxn::new`, and pass it to `MvccReader::new`, instead of adding two set_scan_mode functions.

Reply:
If we pass ScanMode to `MvccReader::new`, it's not necessary to use `gc_reader`?